services/dy-jupyter-extensions/osparc-lab/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2.4'
+services:
+  osparc-lab:
+    image: ${DOCKER_REGISTRY}/osparc-lab:${DOCKER_IMAGE_TAG}
+    runtime: nvidia
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: production
+      args:
+        BASE_IMAGE: ${BASE_IMAGE}
+      labels:
+        io.simcore.key: '{"key": "simcore/services/dynamic/osparc-lab"}'
+        io.simcore.version: '{"version": "${DOCKER_IMAGE_TAG}"}'
+        io.simcore.type: '{"type": "dynamic"}'
+        io.simcore.name: '{"name": "osparc-lab"}'
+        io.simcore.description: '{"description": "oSPARC lab services"}'
+        io.simcore.authors: '{"authors": [{"name": "guidon", "email": "guidon@itis.swiss", "affiliation": "ITIS Foundation"}]}'
+        io.simcore.contact: '{"contact": "guidon@itis.swiss"}'
+        simcore.service.settings: '[
+          {"name": "ports", "type": "int", "value": 8888},
+          {"name": "constraints", "type": "string", "value": ["node.platform.os == linux"]},
+          {"name": "resources", "type": "resources", "value":
+            {
+              "Limits": {
+                "NanoCPUs": 4000000000,
+                "MemoryBytes": 8388608
```

Comment:
Limits < Reservations??

Reply:
Oops. This happens when you mindlessly copy paste